tests/integration/helpers/wrap_kakarot.py

```diff
@@ -68,11 +70,43 @@ async def _wrapped(contract, *args, **kwargs):
                     args=args,
                     kwargs=kwargs,
                 )
+
+            codec = Web3().codec
             types = [o["type"] for o in abi["outputs"]]
             data = bytearray(res.result.return_data)
-            decoded = Web3().codec.decode(types, data)
+            decoded = codec.decode(types, data)
```

Comment:
this can decode a type `felt` ?

Reply:
these types are solidity types actually, used to decode the raw returned data (bytes array / hex string) of the tx (`res = kakarot.execute_at_address(...)`